extensions/sql-migration/src/wizard/skuRecommendation/skuDataCollectionToolbar.ts

```diff
@@ -0,0 +1,121 @@
+		const refreshSKURecommendationButton = view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.REFRESH,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.refresh,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+		// TODO - implement onDidClick and add to disposables
+		return refreshSKURecommendationButton;
+	}
+
+	private createStartPerformanceCollectionButton(view: azdata.ModelView): azdata.ButtonComponent {
+		const startPerformanceCollectionButton = view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.START_PERFORMANCE_COLLECTION,
+				width: 146,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.startDataCollection,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+		// TODO - implement onDidClick and add to disposables
+		return startPerformanceCollectionButton;
+	}
+
+	private createStopPerformanceCollectionButton(view: azdata.ModelView): azdata.ButtonComponent {
+		const stopPerformanceCollectionButton = view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.STOP_PERFORMANCE_COLLECTION,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.stopDataCollection,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+		stopPerformanceCollectionButton.enabled = false;
+		// TODO - implement onDidClick and add to disposables
+		return stopPerformanceCollectionButton;
+	}
+
+	private createImportPerformanceDataButton(view: azdata.ModelView): azdata.ButtonComponent {
+		const importPerformanceDataButton = view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.IMPORT_PERFORMANCE_DATA,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.import,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+		// TODO - implement onDidClick and add to disposables
+		return importPerformanceDataButton;
+	}
+
+	private createRecommendationParametersButton(view: azdata.ModelView): azdata.ButtonComponent {
+		const recommendationParametersButton = view.modelBuilder.button()
+			.withProps({
+				buttonType: azdata.ButtonType.Normal,
+				label: constants.RECOMMENDATION_PARAMETERS,
+				height: 36,
+				iconHeight: 16,
+				iconWidth: 16,
+				iconPath: IconPathHelper.settings,
+				CSSStyles: {
+					...styles.TOOLBAR_CSS
+				}
+			}).component();
+		// TODO - implement onDidClick and add to disposables
+		return recommendationParametersButton;
+	}
+
+	public dispose(): void {
+		// TODO - need to call this at the place where toolbar is initialized
+		this._disposables.forEach(disposable => disposable.dispose());
+		this._disposables.length = 0;
```

Comment:
I would expect the length property of an array to be read-only and not support resizing this way.  While it seems to work, it might be more reliable to assign this._disposables = [] or undefined